modules/gatk4/printsvevidence/main.nf

```diff
@@ -0,0 +1,66 @@
+process GATK4_PRINTSVEVIDENCE {
+    tag "${meta.id}"
+    label 'process_single'
```

Comment:
Is 1 core and 6GB RAM enough for this tool? I'd guess so, as the hard work has presumably been done elsewhere.

Reply:
GATK can only use one core. I don't know for sure if the memory will be fine, but it should be